arlo_server/routes.py

```diff
@@ -1540,6 +1577,14 @@ def handle_401(e):
     )
 
 
+@app.errorhandler(Conflict)
+def handle_409(e):
```

Comment:
Love the appropriate use of lesser-known HTTP error codes 😄

Reply:
❤️ ❤️ ❤️